PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/internal/Settings.java

```diff
@@ -15,7 +15,7 @@
     // connection settings
     public static final String REQUEST_URL_NON_SECURE = "http://prebid.adnxs.com/pbs/v1/auction";
     public static final String REQUEST_URL_SECURE = "https://prebid.adnxs.com/pbs/v1/auction";
-    public static final int REQUEST_TIME_OUR_MILLIS = 500;
+    public static int REQUEST_TIME_OUT_MILLIS = 500;
```

Comment:
Is there convention for all caps here? I thought it was all caps implies constant, and camelCase is a variable, but may be wrong.